dev/NavigationView/NavigationView_ApiTests/NavigationViewTests.cs

```diff
@@ -854,5 +853,23 @@ public void VerifyExpandCollapseChevronVisibility()
             });
         }
 
+        [TestMethod]
+        public void VerifyOverflowButtonToolTip()
+        {
+            RunOnUIThread.Execute(() =>
+            {
+                var navView = new NavigationView();
+                navView.PaneDisplayMode = NavigationViewPaneDisplayMode.Top;
+
+                Content = navView;
+                Content.UpdateLayout();
+
+                var overflowButton = VisualTreeUtils.FindVisualChildByName(navView, "TopNavOverflowButton") as Button;
+                var toolTipObject = ToolTipService.GetToolTip(overflowButton);
+
+                bool testCondition = toolTipObject is ToolTip toolTip && toolTip.Content.Equals("See more");
+                Verify.IsTrue(testCondition, "ToolTip text should have been \"See more\".");
+            });
+        }
```

Comment:
I think testing that a custom tooltip isn't overwritten would be good too.

Reply:
Does Content.UpdateLayout() cause NavigationView.ApplyTemplate() to run or do I need to track this in a new test case?

Reply:
@ranjeshj do you know off the top of your head?

Reply:
ApplyTemplate will be run before running the first layout pass. If you are asking if it will be run again on the next layout (UpdateLayout call), the answer is no.

Reply:
There might be some trouble testing the custom tooltip not being overriden:

Developers today would modify the existing `NavigationViewOverflowButtonStyleWhenPaneOnTop` style to set a custom tooltip like this (you would likely create an exact copy of the existing style and just set the custom tooltip):
```xml
<Style x:Key="NavigationViewOverflowButtonStyleWhenPaneOnTop" TargetType="Button">
    <Setter Property="ToolTipService.ToolTip" Value="Custom tooltip" />
</Style>
```

So in order to test that "Custom tooltip" is not being overriden, my thinking was to create a test NavigationView in code-behind containing that style and then check the tooltip content in an interaction test. However, I noticed that putting this style into the NavigatuonView.Resources scope, or even the Page level scope, does not apply the custom style at all:

```xml
<muxc:NavigationView PaneDisplayMode="Top"">
    <muxc:NavigationView.Resources>
        <Style x:Key="NavigationViewOverflowButtonStyleWhenPaneOnTop" TargetType="Button">
            <Setter Property="Background" Value="Orange" />
            <Setter Property="Foreground" Value="Green" />
            <Setter Property="ToolTipService.ToolTip" Value="Custom tooltip" />
        </Style>
    </muxc:NavigationView.Resources>
</muxc:NavigationView>
```
![image](https://user-images.githubusercontent.com/1398851/89662040-c4d1bc00-d8d3-11ea-9fa0-5c39bfd62d98.png)

If I define that style on the *app* level it is being applied:
![image](https://user-images.githubusercontent.com/1398851/89662232-11b59280-d8d4-11ea-8529-4ff21c97bbef.png) 
![image](https://user-images.githubusercontent.com/1398851/89662255-19753700-d8d4-11ea-9117-eacc65404314.png)

I would expect here that my custom style defined on the page or control level would be picked up (just as how I override theme resources) as the NavigationView overflow button in Top mode is neither in a Popup or flyout.

This immediately poses the problem that if I set that style on the app level, then my existing API test to check the *default* overflow button tooltip will fail as the tooltip content will now by my custom set tooltip.

Thoughts on how to proceed here?

Reply:
This seems like a bug, you should be able to style this from any parent.  If this is blocking the custom tooltip test I think its okay to leave that space uncovered until this new bug is resolved.